docs/source/analyze/mssql.md

```diff
@@ -0,0 +1,55 @@
+---
+title: MS SQL
+description: Check version and connection status
+---
+
+The `MS SQL` analyzer is available to check vesion and connection status of a PostgreSQL database.
+It relies on the data collected by the [MS SQL collector](/collect/mssql/).
+
+The analyzer's outcome `when` clause may be used to evaluate the database connection status or a version range to compare against the running version, and supports standard comparison operators.
+
+## Parameters
+
+**checkName:** Optional name.
+
+**collectorName:** (Recommended) Must match the `collectorName` specified by the mssql collector.
+
+## Outcomes
+
+The `when` value in an outcome of this analyzer contains the connection or version information.
+
+The conditional in the when value supports the following:
+
+**connected:** A boolean representing whether the database is connected.
+Can be compared to a boolean value with the `==` operator.
+
+**version:** A string representing the version of the database.
+Can be compared to a semver string using `<`, `<=`, `>`, `>=`, `==`, `!=`, with the letter 'x' as a version wildcard (10.x).
```

Comment:
...to **an assembly version** string using...

I'm referring to Microsoft's assembly version format => https://learn.microsoft.com/en-us/dotnet/standard/assembly/versioning#assembly-version-number

Reply:
Fixed. Appreciate the review.